roles/container-engine/cri-dockerd/handlers/main.yml

```diff
@@ -3,6 +3,7 @@
   command: /bin/true
   notify:
     - cri-dockerd | reload systemd
+    - reload containerd
```

Comment:
Since this is part of the `cri-containerd` role the task should contain the role name like the other.

Reply:
Thanks. I updated the task name.